lib/storage/mongoose-store.js

```diff
@@ -0,0 +1,34 @@
+var store = require('raincatcher-mongoose-store');
+
+/**
+ *
+ * Connecting to the mongoose store.
+ *
+ * @param connectionString
+ * @returns {*}
+ */
+function connect(connectionString) {
+  return store.connect(connectionString, {});
+}
+
+/**
+ * Disconnecting from the mongoose store. Ensures the mongo connections are closed.
+ */
+function disconnect() {
+  return store.disconnect();
+}
+
+/**
+ *
+ * @param dataSetId
+ * @returns {*}
+ */
+function getCollectionDal(dataSetId) {
+  return store.getDAL(dataSetId);
+}
+
+module.exports = {
+  connect: connect,
+  disconnect: disconnect,
+  getCollectionDal: getCollectionDal
```

Comment:
Same as above. DAL in this case may be DAO, but best to not mention this patterns and simply use storage.
getCollectionStore?